osu.Game/Beatmaps/Drawables/DifficultyIcon.cs

```diff
@@ -23,14 +23,14 @@ public class DifficultyIcon : DifficultyColouredContainer, IHasCustomTooltip
     {
         private readonly RulesetInfo ruleset;
 
-        public DifficultyIcon(BeatmapInfo beatmap, RulesetInfo ruleset = null)
+        public DifficultyIcon(BeatmapInfo beatmap, RulesetInfo ruleset = null, Boolean shouldShowTooltip = false)
```

Comment:
wouldn't it be best to make this true by default, since that's the most common use case?

Reply:
`Boolean` can be simplified into `bool`